src/Flow.cpp

```diff
@@ -1709,15 +1709,15 @@ json_object* Flow::flow2json() {
 
   if(json_info && strcmp(json_info, "{}")) {
     json_object *o;
-    enum json_tokener_error jerr = json_tokener_success;
+    enum json_tokenizer_error jerr = json_tokenizer_success;
```

Comment:
I did worry that this was not local. Fixing this would involve changing the upstream module (if it's willing to do that) and then updating the downstream. I need to figure out where your upstream modules live.

Thanks for dropping it.